src/Frontend/Core/Engine/TemplateModifiers.php

```diff
@@ -87,7 +87,12 @@ public static function getNavigation(
         }
 
         // get HTML
-        $return = (string) Navigation::getNavigationHtml($type, $parentId, $depth, $excludeIds, $template);
+        try {
+            $return = (string) Navigation::getNavigationHTML($type, $parentId, $depth, $excludeIds, $template);
+        } catch (Exception $e) {
+            // if something goes wrong just return as fallback
+            return;
```

Comment:
the phpdoc states `return string`. I would return an empty string here to keep consistency